Bottom: c53db0d13c55029438eeec207218d4c5dfa0ad36
Top:    d7272e53afc556d6e603761582bc58689a59e69c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 20:50:24 +0100

hid/gtk: Remove usage of deprecated gtk_range_set_update_policy()

We only set this to the default value anyway, and the API has been
removed with no replacements.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index cf05e4b..73bada5 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1280,8 +1280,6 @@ ghid_build_pcb_top_window (void)
   ghidgui->v_range =
     gtk_vscrollbar_new (GTK_ADJUSTMENT (ghidgui->v_adjustment));
 
-  gtk_range_set_update_policy (GTK_RANGE (ghidgui->v_range),
-			       GTK_UPDATE_CONTINUOUS);
   gtk_box_pack_start (GTK_BOX (hbox), ghidgui->v_range, FALSE, FALSE, 0);
 
   g_signal_connect (G_OBJECT (ghidgui->v_adjustment), "value_changed",
@@ -1291,8 +1289,6 @@ ghid_build_pcb_top_window (void)
 					      10.0, 10.0, 10.0);
   ghidgui->h_range =
     gtk_hscrollbar_new (GTK_ADJUSTMENT (ghidgui->h_adjustment));
-  gtk_range_set_update_policy (GTK_RANGE (ghidgui->h_range),
-			       GTK_UPDATE_CONTINUOUS);
   gtk_box_pack_start (GTK_BOX (ghidgui->vbox_middle),
                       ghidgui->h_range, FALSE, FALSE, 0);
